internal/api/handler/create_resource_v1alpha1.go

```diff
@@ -0,0 +1,75 @@
+package handler
+
+import (
+	"context"
+
+	"connectrpc.com/connect"
+	"github.com/pkg/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	sigyaml "sigs.k8s.io/yaml"
+
+	"github.com/akuity/kargo/internal/yaml"
+	svcv1alpha1 "github.com/akuity/kargo/pkg/api/service/v1alpha1"
+)
+
+type CreateResourceV1Alpha1Func func(
+	context.Context,
+	*connect.Request[svcv1alpha1.CreateResourceRequest],
+) (*connect.Response[svcv1alpha1.CreateResourceResponse], error)
+
+func CreateResourceV1Alpha1(kc client.Client) CreateResourceV1Alpha1Func {
+	validateProject := newProjectValidator(kc)
+	parseKubernetesManifest := yaml.NewKubernetesManifestParser(kc.Scheme())
+	return func(
+		ctx context.Context,
+		req *connect.Request[svcv1alpha1.CreateResourceRequest],
+	) (*connect.Response[svcv1alpha1.CreateResourceResponse], error) {
+		parsed, err := parseKubernetesManifest(req.Msg.GetManifest())
+		if err != nil {
+			return nil, connect.NewError(connect.CodeInvalidArgument, errors.Wrap(err, "parse manifest"))
+		}
+
+		var res []*svcv1alpha1.CreateResourceResult
+		for _, obj := range parsed {
+			if req.Msg.GetNamespace() != "" {
+				obj.SetNamespace(req.Msg.GetNamespace())
+			}
```

Comment:
It seems here that `req.Msg.GetNamespace()` takes precedence over any value already set in the object. This behavior is different from that of `kubectl` and I think people might find that surprising.

I had to experiment to verify what `kubectl` actually does in this scenario.

`kubectl` surfaces an error for any resource that directly specifies a namespace that differs from the value set via the `--namespace` flag.

It looks like this:

```
the namespace from the provided object "x" does not match the namespace "bar". You must pass '--namespace=foo' to perform this operation.
```

imho, they're using the right approach -- when there's any ambiguity, don't make assumptions.

This applies to the other handlers as well.

Reply:
> I think we can remove support for overriding project (namespace) for now to make things simpler, WDYT?

I think that's sensible. 👍